Indexing_Code_Tour.ipynb

```diff
@@ -458,8 +692,28 @@
     "                                     index_settings=index_settings,\n",
     "                                     index_mapping_config=index_mapping_config)\n",
     "\n",
+    "sample_indexer = SampleOrientedIndexer(metadata_files,\n",
+    "                                       data_files,\n",
+    "                                       es,\n",
+    "                                       \"sample_index_v4\",\n",
+    "                                       \"doc\",\n",
+    "                                       index_settings=index_settings,\n",
+    "                                       index_mapping_config=index_mapping_config)\n",
+    "\n",
+    "project_indexer = ProjectOrientedIndexer(metadata_files,\n",
+    "                                         data_files,\n",
+    "                                         es,\n",
+    "                                         \"project_index_v4\",\n",
+    "                                         \"doc\",\n",
+    "                                         index_settings=index_settings,\n",
+    "                                         index_mapping_config=index_mapping_config)\n",
+    "\n",
     "file_indexer.index(bundle_uuid, bundle_version)\n",
+    "bundle_indexer.index(bundle_uuid, bundle_version)\n",
     "assay_indexer.index(bundle_uuid, bundle_version)\n",
+    "sample_indexer.index(bundle_uuid, bundle_version)\n",
+    "project_indexer.index(bundle_uuid, bundle_version)\n",
+    "\n",
     "print(\"INDEXING DONE\")"
```

Comment:
I think this also be logged at debug. Is the reason that print was being used due to use within the juypter notebook? does logging still work for that, making the desired output available? In any case, this code is intended for execution within a Lambda I think logging is the right thing to do, rather than print.

Reply:
@mikebaumann This print is because it got registered when i ran the notebook. The notebook itself is not meant for executing within the lambda. It's merely there to demo the code when needed. It's really not relevant to the code, but I have it there so I can show people around the output real quick should they want to see how it works.

Reply:
Ah, okay. I didn't notice that this was in the .ipynb file. Thanks!